From 3ccaf35684ae7a3a7541f2afee228905bd3543f6 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Thu, 07 Oct 2021 20:25:39 +0300
Subject: [PATCH] [SW-56241] gaudi2: boot_info: modify base to iccm0

Modify boot_info base address inside ICCM0, now synced
with other projects.

Change-Id: Icefeb93f6b87d78f8aa5e9ccb853bcef149b0b12
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/arch/gaudi2/arm/common.h b/arch/gaudi2/arm/common.h
index d9c1a1d..2653b83 100644
--- a/arch/gaudi2/arm/common.h
+++ b/arch/gaudi2/arm/common.h
@@ -58,8 +58,9 @@
 
 #define HL_INIT_SP_SRAM_RETRIES		50
 
-#define SP_SRAM_BOOT_INFO_OFFSET	0x1800
-#define BOOT_INFO_BASE		(HL_SP_SRAM_BASE + SP_SRAM_BOOT_INFO_OFFSET)
+#define ICCM_BOOT_INFO_OFFSET		0x10000
+#define ARC0_ICCM			0x3FF7000000
+#define BOOT_INFO_BASE			(ARC0_ICCM + ICCM_BOOT_INFO_OFFSET)
 
 /*
  * Set ARCs in halt state before reset
